external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -21,26 +21,26 @@ import java.math.BigDecimal
 import java.sql.{Connection, Date, Timestamp}
 import java.util.Properties
 
-import org.scalatest.Ignore
-
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.types.{BooleanType, ByteType, ShortType, StructType}
 import org.apache.spark.tags.DockerTest
 
 @DockerTest
-@Ignore // AMPLab Jenkins needs to be updated before shared memory works on docker
 class DB2IntegrationSuite extends DockerJDBCIntegrationSuite {
   override val db = new DatabaseOnDocker {
-    override val imageName = "lresende/db2express-c:10.5.0.5-3.10.0"
+    override val imageName = "ibmcom/db2:11.5.0.0a"
     override val env = Map(
       "DB2INST1_PASSWORD" -> "rootpass",
-      "LICENSE" -> "accept"
+      "LICENSE" -> "accept",
+      "DBNAME" -> "foo",
+      "ARCHIVE_LOGS" -> "false",
```

Comment:
`ARCHIVE_LOGS` and `AUTOCONFIG` is needed because it triggers a server side restart (which made the suite fail all the time). Please see the following code snippet extracted from the docker image:
```
configure_db()
{
   dbname=$1

   if [ ${ARCHIVE_LOGS?} = "true" -o "${HADR_ENABLED?}" = "true" ]; then
       # Enabling log archiving does not exit with a zero because of SQL1363W
       enable_log_archiving ${dbname?}
       restart_db2

       if ! back_up ${dbname?}; then
          echo "(!) Failed to back up ${dbname?} database"
       fi
   else
       echo "(*) Log archiving will not be configured as ARCHIVE_LOGS has been set to false. "
   fi
   
   if [ ${AUTOCONFIG?} = "true" -o "${HADR_ENABLED?}" = "true" ]; then
      if ! run_autoconfig ${dbname?}; then
         echo "(!) Failed to automatically configure ${dbname?} database"
      fi
   else
      echo "(*) Instance and database will not be auto configured. AUTOCONFIG has been set to false. "
   fi

   if ! configure_text_search ${dbname?}; then
      echo "(!) Failed to configure ${dbname?} database for text search"
   fi
}
```
I've applied the same in the kerberos suite because until now it was only timing luck that it's not failed.

Reply:
Got it.